pyvista/core/filters.py

```diff
@@ -1129,6 +1129,24 @@ def warp_by_scalar(dataset, scalars=None, factor=1.0, normal=None,
             return
         return output
 
+    def warp_by_vector(dataset, scalars=None, factor=1.0):
```

Comment:
the argument should be `vectors` instead of `scalars` because that's what is required for warping